packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -4,204 +4,549 @@
+                crossAxisAlignment: CrossAxisAlignment.end,
+                children: <Widget>[
+                  Text(
+                    'flex: ${node.flexFactor}',
+                    style: TextStyle(fontWeight: FontWeight.bold),
+                  ),
+                  if (flexFactor == 0 || flexFactor == null)
+                    Text(
+                      'unconstrained ${isRow ? 'horizontal' : 'vertical'}',
+                      style: TextStyle(
+                        color: ThemedColor(
+                          const Color(0xFFD08A29),
+                          Colors.orange.shade700,
+                        ),
+                        fontStyle: FontStyle.italic,
+                      ),
+                      maxLines: 2,
+                      softWrap: true,
+                      overflow: TextOverflow.ellipsis,
+                      textScaleFactor: 0.8,
+                      textAlign: TextAlign.right,
+                    ),
+                ],
               ),
-            ],
+            ),
           ),
-          direction: Axis.vertical,
         ),
-        height: length,
-        width: length * 0.25,
       ),
-      bottom: Container(
-        margin: const EdgeInsets.only(top: 16.0),
-        child: ArrowWrapper.bidirectional(
-          arrowColor: theme.splashColor,
-          arrowStrokeWidth: 1.0,
-          child: Column(
-            children: <Widget>[
-              Text(
-                // TODO(albertusangga): Use real width
-                '${length.round()} px',
-              ),
-              const Text('width'),
-            ],
+    );
+  }
+
+  Widget _buildAxisAlignmentDropdown(Axis axis) {
+    Color color;
+    String axisDescription;
+    List<Object> alignmentEnumEntries;
+    Object selected;
+    if (axis == Axis.horizontal) {
+      color = horizontalColor;
+      axisDescription = properties.horizontalDirectionDescription;
+    } else {
+      color = verticalColor;
+      axisDescription = properties.verticalDirectionDescription;
+    }
+    if (axis == direction) {
+      alignmentEnumEntries = MainAxisAlignment.values;
+      selected = mainAxisAlignment;
+    } else {
+      alignmentEnumEntries = CrossAxisAlignment.values;
+      selected = crossAxisAlignment;
+    }
+    return Container(
+      constraints: const BoxConstraints(maxWidth: kDropdownMaxWidth),
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.start,
+        mainAxisSize: MainAxisSize.min,
+        children: <Widget>[
+          Text(
+            '$axisDescription Alignment: ',
+            textScaleFactor: 1.2,
           ),
-          direction: Axis.horizontal,
-        ),
-        width: length,
+          Container(
+            margin: const EdgeInsets.only(left: 8.0),
+            child: DropdownButton(
+              isExpanded: true,
+              itemHeight: 56,
```

Comment:
why 56? make this a const.

Reply:
dropping this one because seems like default const from flutter (48.0) is enough.

Reply:
+1 for using the defaults when they look good enough :)